pkg/crc/constants/constants_darwin.go

```diff
@@ -7,8 +7,9 @@ import (
 )
 
 const (
-	OcExecutableName   = "oc"
-	TrayExecutableName = "CodeReady Containers.app"
+	OcExecutableName           = "oc"
+	PodmanRemoteExecutableName = "podman-remote"
```

Comment:
On mac the executable name is `podman` instead of `podman-remote`. In linux only it named as `podman-remote`.